doc/source/cluster/kubernetes/examples/rayjob-kueue-priority-scheduling.md

```diff
@@ -0,0 +1,317 @@
+```bash
+$ kubectl create -f ray-job.pytorch-distributed-training.yaml
+rayjob.ray.io/dev-pytorch-text-classifier-8vg2c created
+$ kubectl create -f ray-job.pytorch-distributed-training.yaml
+rayjob.ray.io/dev-pytorch-text-classifier-n5k89 created
+$ kubectl create -f ray-job.pytorch-distributed-training.yaml
+rayjob.ray.io/dev-pytorch-text-classifier-ftcs9 created
+```
+
+Because our `ClusterQueue` was configured with only quotas for 1 RayJob, Kueue will automatically suspend new RayJobs
+until there is sufficient quotas available.
+
+You can also inspect the `ClusterQueue` to see available / used quotas:
+```bash
+$ kubectl get clusterqueue
+NAME            COHORT   PENDING WORKLOADS
+cluster-queue            5
+$ kubectl get clusterqueue  cluster-queue -o yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+...
+...
+...
+status:
+  admittedWorkloads: 1
+  flavorsReservation:
+  - name: default-flavor
+    resources:
+    - borrowed: "0"
+      name: cpu
+      total: "8"
+    - borrowed: "0"
+      name: memory
+      total: 19531250Ki
+    - borrowed: "0"
+      name: nvidia.com/gpu
+      total: "2"
+  flavorsUsage:
+  - name: default-flavor
+    resources:
+    - borrowed: "0"
+      name: cpu
+      total: "8"
+    - borrowed: "0"
+      name: memory
+      total: 19531250Ki
+    - borrowed: "0"
+      name: nvidia.com/gpu
+      total: "2"
+  pendingWorkloads: 5
+  reservingWorkloads: 1
+```
+
+## Deploy a RayJob with Higher Priority
+
+At this point we have multiple RayJobs queued up but only enough quota to run a single RayJob.
+Now we can create a new RayJob with higer priority to preempty the already queued Rayjobs.
+To do this, modify the RayJob with:
+* a label to assign the RayJob to the LocalQueue created earlier.
+* a label to assign the RayJob with the `prod-priority` priority class.
+* Modify the name to indicate this is a job for prod purposes.
+
+```yaml
+metadata:
+  generateName: prod-pytorch-text-classifier-
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+    kueue.x-k8s.io/priority-class: prod-priority
+```
+
+Create the new RayJob:
+```sh
+$ kubectl create -f ray-job.pytorch-distributed-training.yaml
+rayjob.ray.io/prod-pytorch-text-classifier-gkp9b created
+```
+
+Note that higher priority jobs will preempt lower priority jobs when there is not enough quotas for both:
+```bash
+$ kubectl get po 
+NAME                                                      READY   STATUS    RESTARTS   AGE
+prod-pytorch-text-classifier-gkp9b-r9k5r                  1/1     Running   0          5s
+torch-text-classifier-gkp9b-raycluster-s2f65-head-hfvht   1/1     Running   0          35s
+```
+
+## Increasing Cluster Quotas
+
+As a final step, we can control the quotas available by modifying the ClusterQueue.
+Increasing the quotas will signal to Kueue that it can admit more RayJobs. In the event that
+there is no resources available to schcedule new RayJobs, the cluster autoscaler will
```

Comment:
The term "cluster autoscaler" may be very confusing for users in the context of Ray.

```suggestion
there is no resources available to schcedule new RayJob custom resources, the Kubernetes node autoscaler, such as GKE Autopilot, will
```